UNRELEASED.md

```diff
@@ -10,7 +10,7 @@
 
 ### Bug fixes
 
-- Fixed performance of `ResourceItem` due to inclusion of `children` in deep prop comparison within `shouldComponentUpdate` ([#2936](https://github.com/Shopify/polaris-react/pull/2936))
+- Fixed `Banner` `secondaryAction` only rendering if `action` is set ([#2949](https://github.com/Shopify/polaris-react/pull/2949))
```

Comment:
the new change should be a separate line item, instead of overriding the one that is there